packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -84,6 +85,24 @@ export class PathManager {
   getCurrentTagFilePath = (projectPath?: string): string =>
     this.constructPath(projectPath, [PathConstants.AmplifyDirName, PathConstants.CurrentCloudBackendDirName, PathConstants.TagsFileName]);
 
+  getResourceParamatersFilePath = (projectPath: string | undefined, category: string, resourceName: string): string =>
```

Comment:
```suggestion
  getResourceParametersFilePath = (projectPath: string | undefined, category: string, resourceName: string): string =>
```

Reply:
Removed the function altogether as it was not needed.